swagger/v2/component_schemas/ParticipantDeclarationRequest.yml

```diff
@@ -4,4 +4,7 @@ required:
   - data
 properties:
   data:
-    $ref: "#/components/schemas/ParticipantDeclarationDataRequest"
```

Comment:
do we need to do something similar with renaming the `ParticipantDeclarationDataRequest` component in v2/v3?

Reply:
Ah yes, I should have removed that file for v2 and v3 as its redundant now. I'll update 👍